pages/portfolio.html

```diff
@@ -1,12 +1,95 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/grid.css">
+    <title>here is my profile</title>
 </head>
-<body>
-  
-</body>
+
+<div class="wrapper">
+    <div class="grid-container">
+        <header class="grid-header">
+        </header>
+
+        <nav class="grid-top-nav">
+            <ul>
+                <li id="home"><a href="index.html">Home</a></li>
+                <li id="about"><a href="about.html">About</a></li>
+                <li id="portfolio"><a href="portfolio.html">Portfolio</a></li>
+            </ul>
+        </nav>
+
+
+
+        <div class="grid-main">
+            <section class="grid-section">
+                    <img alt="Marjan's profile picture after being an excellent software engineer" src="images/myPic.jpg">
+            </section>
+            <section class="grid-section">
+                <h1 class="portfolio-h1">My Recent Projects:</h1>
+                <div class="portfolio-project-wrapper">
+                    <ol class="p-github-ul">
+                        <li class="p-github-li"><a class="portfolio-github-link" href="https://github.com/Marjan-abm/retro-video-store">Video Store</a></li>
+                        <ul class='p-desc-ul'>
+                            <li>
+                                <p class="project-desc">Once upon a time in ye olden days to watch a movie a person had to head down to their local video store and rent a video. 
+                                    We are going to step into the shoes of this old timey retro video store owner and build for ourselves a Video Store API to handle the back end.
+                                    The full functionality is spelled out further in the "Project Directions" files (linked below). At a high-level, our goal is to create a 
+                                    video store API with the following minimum functionality.</p>
+                                <p class="tech-stack">Tech Stack:
+                                    <li>Python</li>    
+                                </p>
```

Comment:
`<p>` tags can not be used to wrap `<li>` tags.  See the permitted parents section of the MDN for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li